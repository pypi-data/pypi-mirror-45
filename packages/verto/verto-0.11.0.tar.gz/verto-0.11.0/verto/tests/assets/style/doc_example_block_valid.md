This is valid.

{panel}

This is valid.

{panel end}

This is valid.
