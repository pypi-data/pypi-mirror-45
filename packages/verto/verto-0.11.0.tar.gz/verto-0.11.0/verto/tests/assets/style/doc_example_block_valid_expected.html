<p>This is valid.</p>
<p>{panel}</p>
<p>This is valid.</p>
<p>{panel end}</p>
<p>This is valid.</p>
